lib/rules/rest-spread-spacing.js

```diff
@@ -11,13 +11,17 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I think this should be a "style" type rule.

Reply:
Totally agree. I thought I had caught all of the spacing rules.